assets/scss/main.css.scss

```diff
@@ -146,6 +148,11 @@ button.code-button:hover {
   width: 230px;
 }
 
+.usa-nav__close .usa-icon {
```

Comment:
What was this needed for?

Reply:
This was part of the update to LGDS 9.3.0

Reply:
I wouldn't expect you'd need to implement any custom styles beyond what's already included in the design system. Were these missing from the design system  styles?

Reply:
Sorry, I was trying to get that comment in before a meeting.
LGDS 9 removed the close SVG for `<img>` tags, and replaced it with `svg > icon`. It's a drop-in replacement, but we had a custom size on the `img`, and so I'm adding that here, to match what's on `main`.